docs/streams/stream-examples/realtime-updates.md

```diff
@@ -102,33 +267,32 @@ import threading
 import pprint
 import time
 
-# Variables - URLs
-GLOBAL_URL = "play.paas.macrometa.io"
 
-# Variables - DB
-EMAIL = "your@email.com"
-FABRIC = "_system"
+URL = "play.paas.macrometa.io"
+GEO_FABRIC = "_system"
+API_KEY = "my API key" # Change this to your API key
+
 COLLECTION_NAME = "ddos"
 
 # Variables - Data
 data = [
-    {"ip": "10.1.1.1", "action": "block", "rule": "blacklistA"},
-    {"ip": "20.1.1.2", "action": "block", "rule": "blacklistA"},
-    {"ip": "30.1.1.3", "action": "block", "rule": "blacklistB"},
-    {"ip": "40.1.1.4", "action": "block", "rule": "blacklistA"},
-    {"ip": "50.1.1.5", "action": "block", "rule": "blacklistB"},
+    {"ip": "10.1.1.1", "action": "block", "rule": "blocklistA"},
+    {"ip": "20.1.1.2", "action": "block", "rule": "blocklistA"},
+    {"ip": "30.1.1.3", "action": "block", "rule": "blocklistB"},
+    {"ip": "40.1.1.4", "action": "block", "rule": "blocklistA"},
+    {"ip": "50.1.1.5", "action": "block", "rule": "blocklistB"},
 ]
 
 pp = pprint.PrettyPrinter(indent=4)
 
 if __name__ == '__main__':
 
     # Step1: Open connection to GDN. You will be routed to closest region.
-    print(f"\n1. CONNECT: federation: {GLOBAL_URL},  user: {EMAIL}")
-    client = C8Client(protocol='https', host=GLOBAL_URL, port=443, apikey="", geofabric=FABRIC)
+    print(f"\n1. CONNECT: federation: {URL}")
```

Comment:
Do not use "federation" in user-facing documents. In most cases, you can use "server."